packages/components/addon/components/hds/table/index.js

```diff
@@ -5,7 +5,7 @@ import { assert } from '@ember/debug';
 
 const DENSITIES = ['short', 'medium', 'tall'];
 const DEFAULT_DENSITY = 'medium';
-const VALIGNMENTS = ['top', 'middle', 'bottom', 'baseline', 'sub', 'text-top'];
+const VALIGNMENTS = ['top', 'middle'];
```

Comment:
Question: Would it be helpful to include "bottom" as well? ('baseline', 'sub', and 'text-top' seem like they would be much less commonly used but "bottom" seems to have more potential use cases although I don't know if that's true within HashiCorp across various products.)

Reply:
@KristinLBradley great question! "Top" will only really only apply to situations where there is more than one line of content, otherwise it's vertically aligned to the middle (as a browser default).  We added this option because the `hds-button` component is _just_ tall enough to make the cells look a little "off" if the rest of the row only has a single line of content in each cell. 

Then I thought, "oh I wonder if I should add all the acceptable values" but then it was confusing for devs trying to use the component. So I figured I would take out all but the two I absolutely am sure they need, and figured we could add more back in if we had to. WDYT?

Reply:
That makes sense. Thanks for explaining.